website/redirects.js

```diff
@@ -158,6 +158,12 @@ module.exports = [
     destination: "/terraform/cloud-docs/vcs/bitbucket-data-center",
     permanent: true,
   },
+  // Dedicated projects UI
+  {
```

Comment:
Are there any other in-documentation links that refer to this page? I know the redirect handles pushing users to the right page, but I like to update in-doc links too— to catch if there are links to headers that no longer exist, or to update page names if I changed them. 

I saw the [workspace overview page has a link that needs updating](https://terraform-docs-common-dg1e6hkhy-hashicorp.vercel.app/terraform/cloud-docs/workspaces#organize-workspaces-with-projects), but I didn't check elsewhere.

Reply:
Totally forgot about that! Fixed a bunch of these.